cmd/ctl/pkg/create/certificaterequest/certificaterequest.go

```diff
@@ -125,6 +138,14 @@ func (o *Options) Validate(args []string) error {
 		return errors.New("file to store private key must end in '.key'")
 	}
 
+	if o.CertFileName != "" && (len(o.CertFileName) < 4 || o.CertFileName[len(o.CertFileName)-4:] != ".crt") {
+		return errors.New("file to store certificate must end in '.crt'")
+	}
+
+	if !o.FetchCert && o.CertFileName != "" {
+		return errors.New("cannot specify file to store certificate if not waiting for and fetching certificate")
```

Comment:
can we put in the error which flag the user should add to fix this? I usually say this in SRE handbook terms but it might apply here too: if a user has to fix an expired certificate at 3AM they should be able to know exactly how to fix this ;)